docs/README-AWS.md

```diff
@@ -2,68 +2,53 @@
-    * | Key                       | Value  |
-      |---------------------------|--------|
-      |`ELASTIC_APM_ACTIVE`       | `true` |
-      |`ELASTIC_APM_SECRET_TOKEN` | token  |
-      |`ELASTIC_APM_SERVER_URL`	  | url    |
-  * Still in the "Configuration" tab select "Triggers"
-    * You can see an already defined SQS trigger for a queue with the prefix `elastic-serverless-forwarder-continuing-queue-`. This is an internal queue and should not be modified, disabled or removed.
-    * Click on "Add trigger"
-    * From "Trigger configuration" dropdown select "SQS"
-    * In the "SQS queue" field chose the queue or insert the ARN of the queue you want to use as trigger for your Elastic Forwarder for Serverless
-      * The SQS queue you want to use as trigger must have a visibility timeout of 910 seconds, 10 seconds more than the Elastic Forwarder for Serverless Lambda timeout.
-    * Click on "Add"
-    *
 #### Lambda IAM permissions and policies
-Assure the Lambda is given AssumeRole permission to the following `ManagedPolicyArns`:
+A Lambda function has a policy, called an execution role, that grants it permission to access AWS services and resources. Lambda assumes the role when the function is invoked. The role is automatically created when the Function is deployed. The Execution role associated with your function can be seen in the Configuration->Permissions section and by default starts with the name “serverlessrepo-elastic-se-ElasticServerlessForward-”. You can add additional policies to grant minimum permission to the Lambda to be able to use configured continuing SQS queue, S3 buckets, Secrets manager (if using) and replay SQS queue.
+
+Verify the Lambda is given AssumeRole permission to the following `ManagedPolicyArns`. By default this is automatically created:
+`ManagedPolicyArns`:
 * `arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole`
 * `arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole`
 
 On top of this basic permission the following policies must be provided:
-* For the SQS queue resource that's reported in the `SQS_CONTINUE_URL` environment variable the following action must be allowed:
+* For the SQS queues resources that are reported in the `SQS_CONTINUE_URL` and `SQS_REPLAY_URL` environment variable the following action must be allowed:
   * `sqs:SendMessage`
 
 * For every S3 bucket resource that's reported in the `S3_CONFIG_FILE` environment variable the following action must be allowed on the S3 buckets' config file object key:
   * `s3:GetObject`
 
+* For every SQS queue resource that used as triggers of the Lambda the following action must be allowed:
+  * `sqs:GetQueueUrl`
+
+
 * For every S3 bucket resource that SQS queues are receiving notification from used by triggers of the Lambda the following action must be allowed on the S3 buckets' keys:
```

Comment:
I though the above was setup automatically when the user adds the trigger in the UI.
I think we should mention it as a dependency but was thinking that it maybe good to call out more as a verification step rather than something that they need to setup.